docs/src/main/asciidoc/_building.adoc

```diff
@@ -2,25 +2,27 @@
 
 === Cloning the repository on Windows
 
-While cloning this project on *windows*, some files in the git repository may exceed the Windows maximum file path limit of 255 characters, which may
-result in improper/half checked out repository.
+While cloning this project on Windows, some files in the git repository may exceed the Windows maximum file path limit of 255 characters, which may
+result in an incorrectly (probably partially) checked out repository.
 
-* To resolve this the `*core.longPaths=true*` git option should be set:
+* To resolve this issue, you can set the `core.longPaths` attribute to `true` or clone the Spring Cloud Contract repository.
+
+
+The following example shows how to set the `core.longPaths` attribute to `true`:
 
 ```
 git config --system core.longpaths true
 ```
 
 *_OR_*
```

Comment:
Remove this line. (Sorry. I missed it last time.)

Reply:
@Buzzardo Sorry but its not really clear which line are you asking me to remove? Is this the "OR" at line#14.  ?

Reply:
Yes. Please remove the line that consists of `*_OR_*`.

Reply:
@Buzzardo  I have removed that line and also removed a bullet point after it, for the sake of symmetry. Let me know what do you think.